### PR TITLE
Fix placeholder image in bike serializer

### PR DESCRIPTION
**Pull request by jmromer**
Resolves https://app.honeybadger.io/projects/35931/faults/54294084

### CODE REVIEW

**Comment by jmromer** [comment]
Hotfixing

File: app/serializers/bike_v2_serializer.rb [hunk 1]
```diff
@@ -58,13 +58,11 @@ def date_stolen_string
...
-    svg_path =
-      Rails
-        .application
-        .assets["revised/bike_photo_placeholder.svg"]
-        .digest_path
+    assets = Rails.application.assets
+    return if assets.blank?
 
-    "#{ENV["BASE_URL"]}/assets/#{svg_path}"
+    svg_path = assets["revised/bike_photo_placeholder.svg"]&.digest_path
```
Comment (sethherr) [inline app/serializers/bike_v2_serializer.rb]: Should we create some sort of helper for this? Or some sort of caching? Does this create any performance problems?

Comment (sethherr) [reply app/serializers/bike_v2_serializer.rb]: Nevermind, this is resolved by #1223

Comment (jmromer) [reply app/serializers/bike_v2_serializer.rb]: This was an expedient fix — https://github.com/bikeindex/bike_index/pull/1223 takes a better approach.

**Pull request merged**